gulp/modules/LintSpaces.js

```diff
@@ -0,0 +1,82 @@
+var
```

Comment:
Is this plugin forked from `gulp-lintspaces`? Or is it your own? If the former, please also add a comment briefly explaining the purpose of the fork. If the latter, let's chat about the formatting of the file :)